WordPress/Classes/Utility/Media/CachedAsyncImage.swift

```diff
@@ -0,0 +1,74 @@
+import SwiftUI
+
+/// Asynchronous Image View that replicates the public API of `SwiftUI.AsyncImage`.
+/// It uses `ImageDownloader` to fetch and cache the images.
+struct CachedAsyncImage<Content>: View where Content: View {
+    @State private var phase: AsyncImagePhase
+    private let url: URL?
+    private let content: (AsyncImagePhase) -> Content
+    private let imageDownloader: ImageDownloader
+
+    public var body: some View {
+        content(phase)
+            .task(id: url, fetchImage)
+    }
+
+    // MARK: - Initializers
+
+    init(url: URL?) where Content == Image {
+        self.init(url: url) { phase in
+            phase.image ?? Image(uiImage: .init())
```

Comment:
Let's show a default app placeholder here: `Color(uiColor:  .secondarySystemBackground)` and also avoid instantiating empty images.

Reply:
I think you mean Color.DS.Background.secondary 😁?

Resolved in https://github.com/wordpress-mobile/WordPress-iOS/pull/22626/commits/2822e53386b4adfd849d632bd2aedf245dd7f790